### ISSUE TITLE
What's up with sigmoid loss multiplier?

### ISSUE THREAD

**Issue by cinjon** [wzmsltw/BSN-boundary-sensitive-network.pytorch#18]
In this repo, it is .01. In the tensorflow rep, it is .1. Which (of either) did you actually use?

**Comment by wzmsltw** [comment]
This number is used to prevent nan. So both number can work.
